docs/MANUALBUILDING.md

```diff
@@ -0,0 +1,11 @@
+# Manually Building the Installable Eclipse Plugin File  
+
+1. In the package explorer in **Eclipse**, right-click on the `org.eclipse.lsp4jakarta.core.`
```

Comment:
Clarify that users should have their Eclipse workspace set up (and link to instructions for that)

Extra "." at `org.eclipse.lsp4jakarta.core.`?